syft/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -292,6 +293,25 @@ func toSyftPackage(p *spdx.Package) *pkg.Package {
 	return &sP
 }
 
+func parseSPDXLicenses(p *spdx.Package) []pkg.License {
+	licenses := make([]pkg.License, 0)
+
+	// concluded
+	if p.PackageLicenseConcluded != NOASSERTION && p.PackageLicenseConcluded != NONE {
```

Comment:
should check for empty strings as well.

Reply:
Added empty value here for both declared and concluded cases